validator/client/src/test/java/tech/pegasys/teku/validator/client/duties/AttestationProductionDutyTest.java

```diff
@@ -353,7 +354,8 @@ public void shouldCreateAttestationForMultipleValidatorsInDifferentCommittees()
 
   public Validator createValidator() {
     final Signer signer = mock(Signer.class);
-    return new Validator(dataStructureUtil.randomPublicKey(), signer, Optional.empty());
+    return new Validator(
+        dataStructureUtil.randomPublicKey(), signer, new FileBackedGraffitiProvider());
```

Comment:
(nit) For tests that don't directly care about graffiti loading, we can just use `Optional::empty` as the graffiti provider:
```suggestion
        dataStructureUtil.randomPublicKey(), signer, Optional::empty);
```